DatadogSessionReplay/SRSnapshotTests/README.md

```diff
@@ -19,12 +19,12 @@ PNG files are stored in a separate repository. To manage them, use `make` at the
 
 To push new or updated local PNGs to the remote repo:
 ```
-make sr-push-snapshots
+./tools/sr-snapshot-test.sh push
 ```
 
 To pull PNGs from the remote repo:
 ```
-make sr-pull-snapshots
+./tools/sr-snapshot-test.sh pull
```

Comment:
I believe we can still use the make commands, the name only changed (@ncreated commented on this [here](https://github.com/DataDog/dd-sdk-ios/pull/1953#discussion_r1674301733)).

`make sr-snapshots-pull`